lldb/tools/lldb-rpc-gen/RPCCommon.cpp

```diff
@@ -308,6 +264,19 @@ bool lldb_rpc_gen::MethodIsDisallowed(ASTContext &Context,
   return isDisallowed;
 }
 
+// Methods that have a pointer parameter followed by a length param need to be
+// kept track of in RPC. Such methods are annotated in the main SB API using
+// this Clang attribute: [[clang::annotate("lldb-rpc-gen pointer plus len")]].
+// This method checks that a given method has this attribute.
+bool lldb_rpc_gen::MethodContainsPointerPlusLen(CXXMethodDecl *MDecl) {
+  if (!MDecl->hasAttr<clang::AnnotateAttr>())
+    return false;
+
+  auto *annotation = MDecl->getAttr<clang::AnnotateAttr>();
+  return strcmp(annotation->getAnnotation().data(),
```

Comment:
A few points:
1. Avoid strcmp, prefer strncmp if you absolutely need to compare `const char *` types. But in this scenario...
2. `annotation->getAnnotation()` seems to return a `std::string` or an `llvm::StringRef`? I see you calling `.data()` on it, so you can probably just use `==` to compare it against something else.
3. Can you stick `""lldb-rpc-gen pointer plus len"` into a constant somewhere to give it a name?